apps/compact-log-viewer/compact-log-viewer.yml

```
@@ -0,0 +1,12 @@
+name: 'Compact Log Viewer'
+description: 'A cross platform tool to read & query JSON aka CLEF log files created by Serilog'
+website: 'https://github.com/warrenbuckley/Compact-Log-Format-Viewer'
+category: 'Developer Tools'
+repository: 'https://github.com/warrenbuckley/Compact-Log-Format-Viewer'
+keywords:
+    - logviewer
+    - logs
+    - log viewer
+    - clef
+    - compact log event format
```

You intended for this to be a single phrase, correct?

```suggestion
    - 'compact log event format'
```